src/RadixDlt.NetworkGateway.PostgresIntegration/Migrations/MigrationsDbContextModelSnapshot.cs

```diff
@@ -898,6 +872,22 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                 {
                     b.HasBaseType("RadixDlt.NetworkGateway.PostgresIntegration.Models.Entity");
 
+                    b.Property<string>("BlueprintName")
```

Comment:
I'm not sure what happened here and where it came from. I don't see any changes to AccessControllerEntity on that branch, is that some leftover that we forgot to generate migration for?

Reply:
Seems like we must have operate with malformed file previously cause brand-new db migration still results in those changes.